apps/dashboard/src/utils/parseStepVariables.ts

```diff
@@ -1,8 +1,12 @@
+import { Completion } from '@codemirror/autocomplete';
 import type { JSONSchemaDefinition } from '@novu/shared';
+import { DIGEST_VARIABLES, DIGEST_VARIABLES_VALUE_ROOT_PATHS } from '../components/variable/utils/digest-variables';
 
 export interface LiquidVariable {
-  type: 'variable';
+  type: 'variable' | 'digest';
   label: string;
+  boost?: number;
```

Comment:
boost allows the variable to have a higher ranking from other variables which then codemirror uses to sort them in the list
item with boost = 99 will be on the top of the list than item with no boost or lower boost e.g boost = 25